governance/polls/Adjust the WBTC Debt Ceiling and Risk Premium - June 29, 2020

```diff
@@ -0,0 +1,35 @@
+---
+title: Adjust the WBTC Debt Ceiling and Risk Premium - June 29, 2020
+summary: Signal your support for increasing the WBTC Debt Ceiling from 10m to 20m. And increasing the WBTC Risk Premium from 1% to 2%.
+discussion_link: https://forum.makerdao.com/t/2824
+vote_type: Plurality Voting
+options:
+   0: Abstain
+   1: Yes
+   2: No
+---
+# Poll: Adjust the WBTC Debt Ceiling - June 29, 2020
+
+The Maker Governance Community has placed a Governance Poll into the [voting system](https://vote.makerdao.com/polling) which the community can use to increase the WBTC Debt Ceiling from 10m to 20m. And to increase the WBTC Risk Premium from 1% to 2%.
```

Comment:
Purely on the grounds of factual accuracy, it may be better to change this to: "The Governance Facilitators have placed a a Governance Poll into the [voting system](https://vote.makerdao.com/polling) on behalf of the Maker Governance Community. The community can use this poll to express support or opposition to... "

There have been other polls where this isn't super clear, and it bugs me, I'd like everyone to be clear on who is doing what so they know who is responsible for each part of the process.

Reply:
Sure!